tileserver/__init__.py

```diff
@@ -135,6 +135,29 @@ def decode_json_tile_for_layers(tile_data, layer_data):
     return feature_layers
 
 
+def select_layers_from_tile(tile_data, layer_data, coord, format):
```

Comment:
This function also formats the data too, so I'm wondering if we could pick a name for it that reflects that? I'm struggling to come up with a good name though. Perhaps something like `reformat_layers_from_tile`? `reformat_selected_layers`?

Maybe we could also rename `tile_data` to `json_tile_data` to make it more clear as to what the function expects? I'm pretty sure I picked that name in the first place, but it's not as clear to me when reading my own code a second time :)